azbfs/zc_policy_request_log.go

```diff
@@ -22,6 +22,7 @@ type RequestLogOptions struct {
 	// LogWarningIfTryOverThreshold logs a warning if a tried operation takes longer than the specified
 	// duration (-1=no logging; 0=default threshold).
 	LogWarningIfTryOverThreshold time.Duration
+	ForceLog                     bool
```

Comment:
Did you have to make this change in other sdks as well? 
Minor: ForceLog is a misnomer. EnableSyslog/EnableEventLog or something which says what we're doing.

Reply:
Will rename it. 
As for your other question of changes in SDK, yes I am on it.